gallery/plot_transforms_v2_e2e.py

```diff
@@ -0,0 +1,152 @@
+"""
+==================================================
+transforms v2: End-to-end object detection example
+==================================================
+
+Object detection is not supported out of the box by ``torchvision.transforms`` v1, since it only supports images.
+``torchvision.transforms.v2`` enables jointly transforming images, videos, bounding boxes, and masks. This example
+showcases an end-to-end object detection training using the stable ``torchvisio.datasets`` and ``torchvision.models`` as
+well as the new ``torchvision.transforms.v2`` v2 API.
+"""
+
+import pathlib
+from collections import defaultdict
+
+import PIL.Image
+
+import torch
+import torch.utils.data
+
+import torchvision
+
+
+# sphinx_gallery_thumbnail_number = -1
```

Comment:
Is this necessary ? Looks a bit weird once rendered: https://output.circle-artifacts.com/output/job/a6035cf7-ba73-4f96-b92e-1b2aa5b18cfc/artifacts/0/docs/auto_examples/plot_transforms_v2_e2e.html#sphx-glr-auto-examples-plot-transforms-v2-e2e-py

Reply:
This is how we tell sphinx-gallery's to use the (last?) image for the thumbnail in this page
![image](https://user-images.githubusercontent.com/1190450/221128794-65d7d128-e550-4dc6-be07-94aeec1f1049.png)

Reply:
I remembered that there is an option to hide these kind of comments: https://sphinx-gallery.github.io/stable/configuration.html#removing-config-comments Will send a PR